lib/App/MintTag/Remote/GitHub.pm

```diff
@@ -125,12 +125,18 @@ sub _mr_from_raw ($self, $raw) {
     web_url     => $raw->{html_url},
     is_merged   => !! $raw->{merge_commit_sha},
     branch_name => $raw->{head}->{ref},
-    force_push_url => $raw->{head}->{repo}->{ssh_url},
+    force_push_url => $raw->{head}->{repo}->{clone_url},
```

Comment:
Chiming in because my notifications mean I got an email about this: the reason it's not `repo_url` is because I wanted to clarify that it wasn't (say) the URL you'd visit to get to the GitHub repo. In practice, over HTTP these are usually the same thing. GitHub at least redirects the HTTPS clone URL (with `.git) to the canonical web url. (I am delighted to report that I can't easily see what GitLab does right this second.)

Reply:
I agree that it's worth examining whether variable names are sensible but I don't think I want to do a global rename of `force_push_url` here.

It would involve some extra leg work to check if it impacts the deserialization of merge requests and potentially putting in more backwards compatibility checks and then at that point we're probably going to have to start versioning the serialized config...